metagraph/integration_tests/test_query.py

```diff
@@ -38,13 +40,14 @@ def product(graph_types, anno_types):
     result  = []
     for graph in graph_types:
         for anno in anno_types:
-            if graph == 'succinct' or (anno != 'row_diff' and anno != 'row_diff_brwt'):
+            if graph == 'succinct' or (
+                    anno != 'row_diff' and anno != 'row_diff_brwt' and anno != 'row_sparse' and anno != 'row_diff_sparse'):
```

Comment:
Why is `row_sparse` in this list as well?
It should work with other graphs too, right?

Reply:
Ah right, there's nothing special about this one. Fixed.